inbox/api/validation.py

```diff
@@ -28,6 +28,23 @@ def bounded_str(value, key):
     return value
 
 
+def comma_separated_email_list(value, key):
+    addresses = value.split(',')
```

Comment:
Yeah I know, but I've never actually seen that in the wild and it would likely make this change unnecessarily complicated. I'll add a comment and we can change it if someone experiences an issue.